src/eventHandlers/FunctionEnvironmentReloadHandler.ts

```diff
@@ -43,7 +44,7 @@ export class FunctionEnvironmentReloadHandler extends EventHandler<
                 logCategory: LogCategory.System,
             });
             process.chdir(msg.functionAppDirectory);
-            await channel.updatePackageJson(msg.functionAppDirectory);
+            await channel.initalizeApp(msg.functionAppDirectory);

```

Comment:
It looks like `initalizeApp` is the method called from both event handlers so I would create a `initalizeApp.ts` file and put any function it calls from `WorkerChannel.ts` (i.e. `loadEntryPointFile`) also into that file. Then you can create a test file `initalizeApp.test.ts` to test the feature. 

nit: check the spelling of `initalizeApp` and also thoughts on `initialize` vs `startup`? Wondering if we can merge terminology somehow

Reply:
I refactored to a new file "appStartup"